src/web/app/components/response-time-series-chart/response-time-series-chart.component.html

```diff
@@ -5,11 +5,11 @@
     <div>
         <label>Duration:</label>
         <select class="form-control" [ngModel]="model.durationMinutes" (ngModelChange)="setDurationHandler($event)">
-            <option *ngFor="let duration of [1, 5, 10, 15, 30, 60]" [ngValue]="duration">{{ duration + ' min'}}</option>
+            <option *ngFor="let duration of [5, 10, 15, 30, 60]" [ngValue]="duration">{{ 'Past ' + duration + ' mins'}}</option>
```

Comment:
Can we have a more diverse options? Like 15min, 1hr, 3hr, 6hr, 24hr, 2days.
Seeing 5, 10, 15, 30, 60 feels a bit useless, as the graph does not change that drastically, and you can infer 30 minute graph from a 60 minute graph easily.

Reply:
5min, 30min, 1 hour, 3 hours, half a day, 1 day, 1 week, 1 month?

Reply:
^ I think this works

Reply:
ditto, i think the scale works well

Reply:
yes